code/modules/client/preference_setup/loadout/loadout_xeno.dm

```diff
@@ -270,86 +270,222 @@
 	path = /obj/item/clothing/suit/storage/seromi/cloak/jobs/ce
 	allowed_roles = list("Chief Engineer")
 
+/datum/gear/suit/dept/cloak/ce/New() //VS EDIT
```

Comment:
I believe there is already a loadout_xeno_vr you can put these in.

Reply:
I opted to put them here so folks didn't have to uump between files to see the code that gives these loadout items a choice bw Reggie and CitRP versions, but will split them off anyway per your request